lib/assessment.js

```diff
@@ -4,23 +4,45 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
 
-};
+var min = function min(number) {
+numbers = [6,1,3,4,8];
+ var minimum = number[0];
```

Comment:
From here forward, replace all incidences of `number` with `arguments`. The result of that replacement will be a correct min function as specified by the question.